cmd/osbuild-composer/composer.go

```diff
@@ -173,6 +174,12 @@ func (c *Composer) InitAPI(cert, key string, enableTLS bool, enableMTLS bool, en
 		TenantProviderFields: c.config.Koji.JWTTenantProviderFields,
 	}
 
+	// handle experimental image-builder manifest generation option using the
+	// experimentalflags pkg from osbuild/images.
+	if experimentalflags.Bool("image-builder-manifest-generation") {
```

Comment:
(super nitpick) I personally would have combined this with the previous commit that added `ImageBuilderManifestGeneration` to the config but up to you how granular you want this.